extensions/podman/packages/extension/src/extension.ts

```diff
@@ -1390,8 +1391,23 @@ export async function activate(extensionContext: extensionApi.ExtensionContext):
       }
     });
 
+    // register two commands to enable and disable compatibility mode
+    extensionContext.subscriptions.push(
+      extensionApi.commands.registerCommand('podman.disableCompatibilityMode', async () => {
```

Comment:
question: should we had the command to the package.json ? other podman commands are listed in it.

Reply:
we don't need as we don't provide any shorcuts/nice display name etc, being mostly internal